static/json/datadog-agent-ecs.json

```diff
@@ -26,11 +36,23 @@
       "environment": [
         {
           "name": "DD_API_KEY",
-          "value": "INSERT_API_KEY"
+          "value": "<YOUR_API_KEY>"
         },
         {
-      	  "name": "SD_BACKEND",
-	  "value": "docker"
+          "name": "DD_LOGS_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL",
+          "value": "true"
+        },
+        {
+          "name": "DD_PROCESS_AGENT_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "SD_BACKEND",
```

Comment:
this option is not used for agent6 anymore, we can remove it from all agent6 specs

Reply:
fixed :)